impl/sql/mutations/mysql.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build mysql
+
+package mutations
+
+import (
+	_ "github.com/go-sql-driver/mysql" // Set database engine.
+)
+
+var (
+	driverName = "mysql"
+
+	createStmt = []string{
+		`
+	CREATE TABLE IF NOT EXISTS Maps (
+		MapID   VARCHAR(32) NOT NULL,
+		PRIMARY KEY(MapID)
+	);`,
+		`
+	CREATE TABLE IF NOT EXISTS Mutations (
+		MapID    VARCHAR(32)   NOT NULL,
+		Sequence INTEGER       NOT NULL PRIMARY KEY AUTO_INCREMENT,
+                MIndex   VARBINARY(32) NOT NULL,
+		Mutation BLOB          NOT NULL,
+		UNIQUE(MapID, Sequence),
```

Comment:
It looks like this UNIQUE constraint is the only difference between MySQL and MySQLite. 
If so, consider merging the two, since the UNIQUE constraint is not actually doing anything. The PRIMARY KEY and NOT NULL won't allow duplicates anyhow.